CONTRIBUTING_CODE.md

```diff
@@ -2,25 +2,35 @@
 
 To modify existing Great Expectations code, you complete the following tasks:
 
-- [Fork and clone the Great Expectations repository](#fork-and-clone-the-great-expectations-repository)
-
-- [Install Python dependencies](#install-python-dependencies)
-
-- [Create a virtual environment (optional)](#create-a-virtual-environment-optional)
-
-- [Install dependencies from requirements-dev.txt](#install-dependencies-from-requirements-devtxt)
-
-- [Install great_expectations](#install-great_expectations)
-
-- [Configure backends for testing (optional)](#configure-backends-for-testing-optional)
-
-- [Test code changes](#test-code-changes)
-
-- [Test performance](#test-performance)
-
-- [Submit a pull request](#submit-a-pull-request)
-
-To discuss your code change before you implement it, join the [Great Expectations Slack community](https://greatexpectations.io/slack) and make your suggestion in the [#contributing](https://greatexpectationstalk.slack.com/archives/CV828B2UX) channel. 
+- [Contribute a code change](#contribute-a-code-change)
+  - [Prerequisites](#prerequisites)
+  - [Fork and clone the Great Expectations repository](#fork-and-clone-the-great-expectations-repository)
+  - [Install Python dependencies](#install-python-dependencies)
+  - [Create a virtual environment (optional)](#create-a-virtual-environment-optional)
+    - [Python](#python)
+    - [Anaconda](#anaconda)
+  - [Install dependencies from requirements-dev.txt](#install-dependencies-from-requirements-devtxt)
```

Comment:
Looks like this was re-generated.

Reply:
This looks out of sync with develop, is this maybe a merge conflict resolution issue? https://github.com/great-expectations/great_expectations/blob/develop/CONTRIBUTING_CODE.md